### PR TITLE
Moved scripts into their corresponding folders.

### PR DESCRIPTION
**Pull request by VahidooX**
Signed-off-by: VahidooX <vnoroozi@nvidia.com>

### CODE REVIEW

File: docs/sources/source/nlp/bert_pretraining.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Download Corpus
 The training corpus can be either raw text where data preprocessing is done on the fly or an already preprocessed data set. In the following we will give examples for both.
 To showcase how to train on raw text data, we will be using the very small WikiText-2 dataset :cite:`nlp-bert-merity2016pointer`.
 
-To download the dataset, run the script ``examples/nlp/scripts/get_wkt2.sh download_dir``. After downloading and unzipping, the folder is located at `download_dir` and should include 3 files that look like this:
+To download the dataset, run the script ``examples/nlp/language_modeling/get_wkt2.sh download_dir``. After downloading and unzipping, the folder is located at `download_dir` and should include 3 files that look like this:
```
Comment (ekmb) [inline docs/sources/source/nlp/bert_pretraining.rst]: get_wkt2.sh is also referenced in the lm_bert_dataset and lm_transformer_dataset please update the path

File: docs/sources/source/nlp/ner.rst [hunk 2]
```diff
@@ -46,7 +46,7 @@ Each line of the text.txt file contains text sequences, where words are separate
 You can use `this`_ to convert CoNLL-2003 dataset to the format required for training.
 
 
-.. _this: https://github.com/NVIDIA/NeMo/blob/master/scripts/convert_iob_format_to_token_classification_format.py
+.. _this: https://github.com/NVIDIA/NeMo/blob/master/token_classification/convert_iob_format_to_token_classification_format.py
```
Comment (ekmb) [inline docs/sources/source/nlp/ner.rst]: new file name?

File: examples/nlp/token_classification/NERWithBERT.ipynb [hunk 3]
```diff
@@ -24,7 +24,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can download data from [here](https://github.com/kyzhouhzau/BERT-NER/tree/master/data) and use [this](https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/scripts/convert_iob_format_to_token_classification_format.py) script to preprocess it."
+    "You can download data from [here](https://github.com/kyzhouhzau/BERT-NER/tree/master/data) and use [this](https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/token_classification/convert_iob_format_to_token_classification_format.py) script to preprocess it."
```
[the remark this answers is not in the document]
Comment (VahidooX) [reply examples/nlp/token_classification/NERWithBERT.ipynb]: Done. Please check it to see if it is right.

File: examples/nlp/token_classification/PunctuationWithBERT.ipynb [hunk 4]
```diff
@@ -50,7 +50,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this notebook we're going to use a subset of English examples from the [Tatoeba collection of sentences](https://tatoeba.org/eng), set NUM_SAMPLES=-1 and consider including other datasets to improve the performance of the model. Use [NeMo/nemo/collections/nlp/data/scripts/get_tatoeba_data.py](https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/scripts/get_tatoeba.py) to download and preprocess the Tatoeba data."
+    "In this notebook we're going to use a subset of English examples from the [Tatoeba collection of sentences](https://tatoeba.org/eng), set NUM_SAMPLES=-1 and consider including other datasets to improve the performance of the model. Use [NeMo/nemo/collections/nlp/data/token_classification/get_tatoeba_data.py](https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/scripts/get_tatoeba.py) to download and preprocess the Tatoeba data."
```
[the remark this answers is not in the document]
Comment (VahidooX) [reply examples/nlp/token_classification/PunctuationWithBERT.ipynb]: Done. Please check it to see if it is right.

File: docs/sources/source/nlp/bert_pretraining.rst (hunk 1 above)
Comment (VahidooX) [reply docs/sources/source/nlp/bert_pretraining.rst]: I think I have updated them already. Are the still outdated?

File: docs/sources/source/nlp/ner.rst (hunk 2 above)
Comment (VahidooX) [reply docs/sources/source/nlp/ner.rst]: Done. Please check it to see if it is right.

File: examples/nlp/token_classification/PunctuationWithBERT.ipynb [hunk 5]
```diff
@@ -50,7 +50,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this notebook we're going to use a subset of English examples from the [Tatoeba collection of sentences](https://tatoeba.org/eng), set NUM_SAMPLES=-1 and consider including other datasets to improve the performance of the model. Use [NeMo/nemo/collections/nlp/data/scripts/get_tatoeba_data.py](https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/scripts/get_tatoeba.py) to download and preprocess the Tatoeba data."
+    "In this notebook we're going to use a subset of English examples from the [Tatoeba collection of sentences](https://tatoeba.org/eng), set NUM_SAMPLES=-1 and consider including other datasets to improve the performance of the model. Use [NeMo/examples/nlp/token_classification/get_tatoeba_data.py](https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/scripts/get_tatoeba.py) to download and preprocess the Tatoeba data."
```
Comment (ekmb) [inline examples/nlp/token_classification/PunctuationWithBERT.ipynb]: this one is still get_tatoeba_data.py not get_tatoeba.py

File: docs/sources/source/nlp/bert_pretraining.rst (hunk 1 above)
Comment (VahidooX) [reply docs/sources/source/nlp/bert_pretraining.rst]: Done.

File: examples/nlp/token_classification/PunctuationWithBERT.ipynb (hunk 5 above)
Comment (VahidooX) [reply examples/nlp/token_classification/PunctuationWithBERT.ipynb]: Changed to get_tatoeba_data.py

**Comment by VahidooX** [comment]
> Ok, so just to make sure that I understand it properly: the essence of that change is to simplify the directory structure and have scripts that e.g. download or preprocess data in the same folder as the examples that train models?

Yes, each script is specific to a task. So moved it there to have a more organized and easy access structure.

**Pull request merged**